src/app/loginComponents/accounts/internal/change-username/change-username.component.scss

```diff
@@ -1,20 +1,61 @@
+@use '@angular/material' as mat;
+
+@import '/src/materialColorScheme.scss';
+
 .change-username-form {
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
 }
 
-.change-username-form-buttons {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+mat-form-field {
+  width: 60%;
 }
 
-#updateUsername {
-  width: 200px;
+mat-error {
+  line-height: 2rem;
 }
 
 .vertical-center {
   display: flex;
   align-items: center;
 }
+
+// Form field inner
+:host ::ng-deep .mat-form-field-wrapper .mat-form-field-infix {
+  border: 0rem;
+  padding: 0rem;
+}
+
+//Form field box
+:host ::ng-deep .mat-form-field-wrapper {
+  margin: 0rem;
+  padding-bottom: 0.8rem;
+  line-height: 4rem;
+}
+
+// Form field label
+:host ::ng-deep .mat-form-field-wrapper .mat-form-field-flex .mat-form-field-infix .mat-form-field-label-wrapper .mat-form-field-label {
+  top: 1.8rem;
+}
+
+// Form field text animation re-implementation
+:host ::ng-deep .mat-form-field-appearance-outline.mat-form-field-can-float.mat-form-field-should-float .mat-form-field-label {
+  transform: translateY(-1.1em) scale(0.75);
+  width: 133.33333%;
+  top: 1.84375em;
+}
+
+// Purple outline and text
```

Comment:
I think I've seen this purple outline and text form fields in other zeplin pages. If all form fields look like this in zeplin, it would be better if these form field styles are in the global style sheet.

Note that if you move them to the global sheet, some of the padding and line height styles you added may make some of the current forms look weird... One example is the registering a custom remote workflow form when I tried it locally.

Reply:
hmm you think we should add all of the form field styles (like changing the form field box height and padding) or just the style for the purple outline and text? I don't know about changing the form field heights because I'm worried about consistency - with `mat-form-field`, `form` and `input`, and drop downs

Also what's the weird form look like? I'm putting the style in the global style sheet and it's actually not doing anything...

Reply:
Yeah you're right, the safest option would just be moving the style for the purple outline and text because it seems like alot of other things would be affected.

> Also what's the weird form look like? I'm putting the style in the global style sheet and it's actually not doing anything...
I had to remove `:host ::ng-deep` for it to work globally and add `!important` to the color I think

Example of weird form with all styles applied
![image](https://user-images.githubusercontent.com/25287123/165628593-ea9ac50f-c9a3-4e08-8daf-fa485e0143ea.png)

A good place to check how the global style applied is on the search page 
![image](https://user-images.githubusercontent.com/25287123/165630824-efd61740-aaaf-473f-a558-aac7f4ba54c8.png)

Note that the form field text and border shouldn't be purple when it's not focused: [search form field in zeplin](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/5fc690e874e0c807afeb299b) (I think you might just need to add `.mat-focused` like you did on line 52)

Reply:
okay great! Thanks kathy, I'll work on adding that